Sources/StreamChat/Database/DTOs/CurrentUserDTO.swift

```diff
@@ -91,6 +92,8 @@ extension NSManagedObjectContext: CurrentUserDatabaseSession {
 
         let mutedUsers = try payload.mutedUsers.map { try saveUser(payload: $0.mutedUser) }
         dto.mutedUsers = Set(mutedUsers)
+        
+        dto.blockedUserIds = currentUser?.blockedUserIds ?? []
```

Comment:
`payload.blockedUserIds`?

Reply:
Yeah, also noticed this as well. Already fixed 🤠